packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -102,7 +103,7 @@ const ModalManager = () => {
         enableCFDPasswordModal();
     };
 
-    const existing_accounts_data = (acc_type: 'synthetic' | 'financial') => {
+    const existing_accounts_data = (acc_type: 'synthetic' | 'financial' | 'all') => {
```

Comment:
```suggestion
    const existing_accounts_data = (acc_type: TTradingPlatformAvailableAccount['market_type']) => {
```
@hamza-deriv We shouldn't be maintaining the same types everywhere in the code, We should infer instead.

Reply:
We can use this suggestion but have to add `| 'synthetic'` because synthetic is refered as `gaming` in some places

Reply:
Done